blocks/api/registration.js

```diff
@@ -330,3 +330,25 @@ export const getChildBlockNames = ( blockName ) => {
 export const hasChildBlocks = ( blockName ) => {
 	return select( 'core/blocks' ).hasChildBlocks( blockName );
 };
+
+/**
+ * Registers a new block style variation for the given block
+ *
+ * @param {string} blockName      Block type name.
+ * @param {Object} styleVariation Block style variation.
```

Comment:
Should we document what properties this object has?

```
@param {Object} styleVariation Object containing `name` which is the class name applied to the block and `label` which identifies the variation to the user.
```

Reply:
Yes we should 😅